### ISSUE TITLE
Application Task 7 Part 6 Join 

### ISSUE THREAD

**Issue by schmimol** [STA518-Spring20/discussion#59]
I feel like I’ve tried every combination of joining the two data sets by state and trying not to use division/region, but no matter what I’ve tried the map still doesn’t color the US by division. It still looks black in the background. So, I’m just confused about what the join should look like I guess.

**Comment by dykesb** [comment]
One way that I thought of doing this is to have two datasets: 
1. the dennys/laquintas/divison dataset, and 
2. the states shapefile dataset (that I recommended in #53 ).

The states shapefile dataset also will need to have the division information joined to it. Then, in the `geom_polygon()` layer, you can `fill` based on the divisions.

**Comment by chycinsa** [comment]
I'm using fill based on decisions, but only Hawaii and Alaska are properly colored. The mainland US is showing up as grey, or NA, even though after the join there are no NAs in my table

**Comment by Rand3rino** [comment]
I used both ```map_data("state")``` and ```map_data("world") %>% filter(region == "USA")``` for this section. The first one has the continental states + DC while the second includes Alaska and Hawaii. 

You can filter the second dataset to only include Alaska and Hawaii and append the datasets together for the full 50 states.

**Comment by hendrmat** [comment]
Sorry I'm joining this discussion a little late.  If anyone is still having trouble, for joining the states shapefile dataset to the division information, I ended up merging the division and state information.  Afterward, I created another column for the division info with the lowercase version of the state names.  I was then able to do the join (I was having a really hard time with this part for quite a while) as the data would not join smoothly beforehand.  I know there are much more elegant solutions out there than what I did, though.
